README

```diff
@@ -1,23 +1,10 @@
-This module generates image quality benchmarks for a emission tomography
+This module generates image quality benchmarks for a tomography
 simulation with optional Poisson noise using TomoPy.
 
-You can run the benchmark using the provided run-benchmarking.sh. The benchmarks
-are divded into three main scripts. project.py which will generate the simulated
-data. reconstruct.py which will use one dataset to do reconstruction for all
-of the TomoPy reconstruction algorithms and rate the results using MS-SSIM.
-summarize.py which will collect the image quality data into a JSON formatted
-text file and pull in data from the TiMemory benchmarks if they are in the
-same folders.
+You can run the benchmark using the provided run-benchmarking-cpu to run the CPU only functions or run-benchmarking-gpu to run the GPU only functions. The benchmarks are divided into three main scripts. project.py which will generate the simulated data. reconstruct.py which will use one dataset to do reconstruction for all of the TomoPy reconstruction algorithms and rate the results using MS-SSIM. summarize.py which will collect the image quality data into a JSON formatted text file. The results will be saved in a folder labeled tomopy.github.io/(CURRENT DATE). 
```

Comment:
> The results will be saved in a folder labeled tomopy.github.io/(CURRENT DATE)/cpu for CPU-based reconstruction algorithms and tomopy.github.io/(CURRENT DATE)/gpu for GPU-based reconstruction algorithms

This is not correct. The results are saved in any directory that the user specifies with the `--output-dir` parameter. Only the two example scripts: `run-benchmarking-*.sh` use those directories. This first section of the README is summary of how to use the benchmarking package. If you want to document something about the specific Azure jobs that we run, then you should start a separate section to describe it.